src/manage/getAccessGrantAll.ts

```diff
@@ -83,6 +88,7 @@ async function getAccessGrantAll(
     vcShape as Partial<VerifiableCredential>,
     {
       fetch: sessionFetch,
+      includeExpiredVc: options.includeExpiredGrants,
```

Comment:
```suggestion
      includeExpiredVc: options.includeExpired,
```

(I would probably use also includeExpired at the low-level library too)